web/client/components/geostory/common/map/Controls.jsx

```diff
@@ -86,7 +86,7 @@ export const Controls = ({
             />
             <FeatureInfoFormatSelector
                 disabled={!options.mapInfoControl}
-                infoFormat={options.mapOptions && options.mapOptions.mapInfoFormat || "text/plain"}
+                infoFormat={options.mapOptions && options.mapOptions.mapInfoFormat || "application/json"}
```

Comment:
why did you changed the default format? 
`text/plain` is compatible with all servers. application/json is available only on GeoServer, so in some cases it may not work.